accounts/abi/unpack.go

```diff
@@ -93,6 +93,17 @@ func readFixedBytes(t Type, word []byte) (interface{}, error) {
 
 }
 
+func getDeepSizeForType(t *Type) int {
```

Comment:
Nitpicking here: I would prefer getTotalArraySize rather than "deep" size.

Reply:
It's like a deep copy; it repeats the same operation up to the deepest array level. Total array size would work too, but it misses the aspect of nesting imho.